ansible/roles/dev-desktop/files/github_app/readme.md

```diff
@@ -0,0 +1,53 @@
+These instructions are for server-side setup and debugging.
+The user only needs to be directed to the app installation URL
+and everything should just work for them.
+
+We're using the python github library for all github operations.
+You can find the docs at https://pygithub.readthedocs.io/en/latest/introduction.html
+
+# How to setup an App
+
+1. Go to https://github.com/settings/apps
+2. New Github App
+3. Fill out metadata (name and url)
+4. disable WebHook checkbox
+5. Set `Contents - Repository contents, commits, branches, downloads, releases, and merges.` to read/write
```

Comment:
The workflows permission also needs to be granted, otherwise it's not possible to push GitHub Actions workflow files.